js/actions/event-listeners.js

```diff
@@ -1,7 +1,8 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-/* global window gThreadClient setNamedTimeout */
+/* global window gThreadClient setNamedTimeout services EVENTS */
+/* eslint no-shadow: 0  */
```

Comment:
helps with a dispatch call on line 34, where dispatch is shadowing the outer define